src/TextArea.tsx

```diff
@@ -0,0 +1,164 @@
+import styled, {css} from 'styled-components'
+import React, {TextareaHTMLAttributes, ReactElement} from 'react'
+import {FormValidationStatus} from './utils/types/FormValidationStatus'
+import sx, {SxProp} from './sx'
+import {get} from './constants'
+
+export type TextAreaProps = {
+  /**
+   * Apply inactive visual appearance to the TextArea
+   */
+  disabled?: boolean
+  /**
+   * Indicates whether the TextArea must be checked
+   */
+  required?: boolean
+  /**
+   * Indicates whether the TextArea validation state
+   */
+  validationStatus?: FormValidationStatus
+  /**
+   * Block
+   */
+  block?: boolean
+  /**
+   * Allows resizing of the textarea
+   */
+  resize?: 'none' | 'both' | 'horizontal' | 'vertical'
+  /**
+   * Applies a custom width to the textarea
+   */
+  width?: number
+  /**
+   * Applies a custom height to the textarea
+   */
+  height?: number
+} & TextareaHTMLAttributes<HTMLTextAreaElement> &
+  SxProp
+
+const StyledTextArea = styled.textarea<TextAreaProps>`
+  min-height: 124px;
+  font-family: inherit;
+  font-size: ${get('fontSizes.1')};
+  line-height: 20px;
```

Comment:
Should we have primitives for line-height? cc @colebemis 

(feedback isn't specifically relevant to this PR)

Reply:
```suggestion
  line-height: ${get('lineHeights.condensed')};
```

We have lineHeights available in the theme (These should be lifted into primitives eventually)

Reply:
```
"lineHeights": {
    "condensedUltra": 1,
    "condensed": 1.25,
    "default": 1.5
  },
```